### PR TITLE
DOMA-2503 Fix tables scrolling in low descktop resolution

### PR DESCRIPTION
**Pull request by VKislov**
Screenshot from 2022-04-21 17-49-17
Screenshot from 2022-04-21 17-50-49
Screenshot from 2022-04-21 17-53-37

### CODE REVIEW

File: apps/condo/domains/common/components/LayoutContext.tsx [hunk 1]
```diff
@@ -34,6 +35,7 @@ export const LayoutContextProvider: React.FC = (props) => {
     }
 
     const isSmall = (breakpoints.md || breakpoints.xs || breakpoints.sm) && !breakpoints.lg
+    const tableScrollInLowResolution = (breakpoints.md || breakpoints.xs || breakpoints.sm) && !breakpoints.xl
```
Comment (akarjakin) [inline apps/condo/domains/common/components/LayoutContext.tsx:38]: maybe `lowResScroll` ? It's anyway used within tables, so we can make it a bit shorter.

**Review by akarjakin — APPROVED** [review]

File: apps/condo/domains/common/components/LayoutContext.tsx (hunk 1 above)
Comment (VKislov) [reply apps/condo/domains/common/components/LayoutContext.tsx]: Fixed

**Review by sitozzz — APPROVED** [review]
LGTM!

File: apps/condo/domains/common/components/LayoutContext.tsx [hunk 2]
```diff
@@ -9,6 +9,7 @@ const { useBreakpoint } = Grid
 interface ILayoutContext {
     isMobile?: boolean
     isSmall?: boolean
+    lowResScroll?: boolean
```
Comment (AleX83Xpert) [inline apps/condo/domains/common/components/LayoutContext.tsx:12]: This variable is defined within `LayoutContext`, so it's good to know without debugging what this variable using for.
Also, this is a boolean variable. A good practice to name such variables starting with `is`, `has`, `should`.
As for this particular one, I suggest `shouldTableScroll`.

File: apps/condo/domains/common/components/LayoutContext.tsx (hunk 1 above)
Comment (AleX83Xpert) [reply apps/condo/domains/common/components/LayoutContext.tsx]: I disagree. See my [comment below](#discussion_r855346923).

File: apps/condo/domains/common/components/LayoutContext.tsx [hunk 3]
```diff
@@ -34,6 +35,7 @@ export const LayoutContextProvider: React.FC = (props) => {
     }
 
     const isSmall = (breakpoints.md || breakpoints.xs || breakpoints.sm) && !breakpoints.lg
+    const shouldTableScroll = (breakpoints.md || breakpoints.xs || breakpoints.sm) && !breakpoints.xl
```
Comment (pahaz) [inline apps/condo/domains/common/components/LayoutContext.tsx:38]: why you don't use `isSmall` in this case?

**Pull request merged**